net/bulk_emulation/ut/bulk_mem_ut.c

```diff
@@ -298,7 +298,7 @@ static void test_failure(void)
 	M0_UT_ASSERT(d1tm1.ntm_state == M0_NET_TM_STARTED);
 	M0_UT_ASSERT(strcmp(d1tm1.ntm_ep->nep_addr, "127.0.0.1:10") == 0);
 	M0_SET0(&d1nb1);
-	M0_UT_ASSERT(!m0_bufvec_alloc(&d1nb1.nb_buffer, 4, 10));
+	M0_UT_ASSERT(!m0_bufvec_alloc(&d1nb1.nb_buffer, 1, 40));
```

Comment:
Please explain the reason of this change.

Reply:
In case of libfabric, for RPC max seg_nr = 1 and max segment size = 1MB